charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml

```diff
@@ -45,18 +60,34 @@ spec:
       labels:
         app: helm-repository  # pods have this label, and Service and Deployment depend on it
     spec:
+      initContainers:
+      - name: init-container
+        image: "{{ .Values.helmChartsImage.repository }}:{{ default $.Chart.AppVersion .Values.helmChartsImage.tag }}"
+        command: ["/bin/sh", "-c", "cp /charts/*.tar /helm-charts"]
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
```

Comment:
`volumeMount` for the initContainer is missing

Reply:
nice catch here